Examples/Tests/plasma_lens/inputs_test_3d_plasma_lens.py

```diff
@@ -0,0 +1,77 @@
+from pywarpx import (
+    Bucket,
+    Diagnostics,
+    Particles,
+    algo,
+    amr,
+    boundary,
+    diagnostics,
+    geometry,
+    particles,
+    picmi,
+    warpx,
+)
+
+clight = picmi.constants.c
+m_e = picmi.constants.m_e
+q_e = picmi.constants.q_e
+
+# Maximum number of time steps
+max_step = 84
+
+# number of grid points
+amr.n_cell = 16, 16, 16
+
+amr.max_level = 0
+
+# Geometry
+geometry.dims = "3"
+geometry.prob_lo = -1.0, -1.0, 0.0  # physical domain
+geometry.prob_hi = 1.0, 1.0, 2.0
+
+boundary.field_lo = "pec", "pec", "pec"
+boundary.field_hi = "pec", "pec", "pec"
+boundary.particle_lo = "absorbing", "absorbing", "absorbing"
+boundary.particle_hi = "absorbing", "absorbing", "absorbing"
+
+# Algorithms
+algo.particle_shape = 1
+warpx.cfl = 0.7
+
+vel_z = 0.5 * clight
+
+# particles
+particles.species_names = ("electrons",)
+
+electrons = Particles.newspecies("electrons")
+electrons.charge = -q_e
```

Comment:
This will be slightly different behavior since the `-q_e` will be evaluated in the input file rather than in the C++ parameter parsing. This could also be
```electrons.charge = "-q_e"```
which would be evaluated in C++.